gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Dec 12 14:16:10 EST 2019
```

Comment:
Did you run `gradle wrapper` or something? Lots of extra diff

Reply:
Yes I ran gradle wrapper for the version we're using